src/content/docs/accounts/accounts-billing/account-structure/mastersub-account-structure.mdx

```diff
@@ -16,15 +16,27 @@ redirects:
   - /docs/accounts-partnerships/accounts/account-setup/manage-apps-or-users-sub-accounts
   - /docs/accounts-partnerships/install-new-relic/account-setup/manage-apps-or-users-sub-accounts
   - /docs/accounts/install-new-relic/account-setup/manage-apps-or-users-sub-accounts
+  - /docs/accounts/install-new-relic/account-setup/use-multiple-accounts/
+  - /docs/accounts-partnerships/accounts/account-setup/creating-multiple-accounts
+  - /docs/accounts-partnerships/accounts/account-setup/create-multiple-accounts
+  - /docs/accounts-partnerships/accounts/account-setup/use-multiple-accounts
+  - /docs/accounts-partnerships/install-new-relic/account-setup/use-multiple-accounts  
 ---
 
 If you have Pro or Enterprise tier, you can create a more complex account structure with multiple accounts in one organization.
 
+## Requirements [#requirements]
+
+How you manage accounts and users differs based on which account/user model your organization is on. To learn more about the account/user structures, see [User models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models). The major differences are: 
```

Comment:
Although the User models link seems to work, it does look like it's formatted oddly. Should it be: `/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models`

Reply:
are you referring to the / or lack of it before anchor link? from my undrestanding that works either way.

Reply:
the / before the anchor link. It just looks weird to me, but it worked when I checked it so it's probably all right.